ansible/group_vars/nimbus.kiln.yml

```diff
@@ -38,12 +38,15 @@ geth_websocket_port: '{{ 9546 + idx|int + 1 }}'
 beacon_node_service_name: 'beacon-node-{{ beacon_node_network }}-devauth-{{ "%02d"|format(idx|int+1) }}'
 beacon_node_network: 'kiln'
 beacon_node_proposer_boosting_debug: true
-# FIXME: Pinned because bf127920 breaks the testnet config.
-beacon_node_repo_branch: 'a35c5f83a8f7880165e19784f73463663db9fdd8'
+beacon_node_repo_branch: 'kiln-dev-auth'
```

Comment:
`kiln-dev-auth` head is the branch to use for kiln network. The issue highlighted in comment has been fixed:
https://github.com/status-im/nimbus-eth2/blob/kiln-dev-auth/beacon_chain/networking/network_metadata.nim#L208

`kiln-dev-auth` also supports light client data config:
https://github.com/status-im/nimbus-eth2/blob/kiln-dev-auth/beacon_chain/conf.nim#L429-L440